doc/whats-new.rst

```diff
@@ -49,6 +49,8 @@ Deprecations
 Bug fixes
 ~~~~~~~~~
 
+- Proper plotting with BoundaryNorm type argument in :py:meth:`DataArray.plot(norm=matplotlib.colors.BoundaryNorm)`. (:issue:`4061`, :issue:`7014`,:pull:`7553`)
```

Comment:
```suggestion
- Proper plotting when passing :py:class:`~matplotlib.colors.BoundaryNorm` type argument in :py:meth:`DataArray.plot`. (:issue:`4061`, :issue:`7014`,:pull:`7553`)
```

Reply:
Also, merge main and move to 2023.04 section